src/extending-phpunit.rst

```diff
@@ -6,100 +6,175 @@
 Extending PHPUnit
 *****************
 
+.. _extending-phpunit.enhancing-concrete-test-cases:
+
+Enhancing concrete test cases
+=============================
+
+You can extend PHPUnit by enhancing concrete test cases with methods that add functionality.
+
+For example, you may use an assertion in a concrete test case to assert that a value created by the system under test matches a regular expression.
+
+.. literalinclude:: examples/extending-phpunit/OrderIdGeneratorTest.php
+   :caption: A concrete test case using a default assertion
```

Comment:
When I render the documentation in a local environment, the captions are not numbered. I do not know how to achieve this effect.

For example, see https://docs.phpunit.de/en/10.1/writing-tests-for-phpunit.html#writing-tests-for-phpunit-output-examples-outputtest-php:

<img width="1305" alt="CleanShot 2023-05-29 at 14 01 11@2x" src="https://github.com/sebastianbergmann/phpunit-documentation-english/assets/605483/ea4c5959-4a66-4d00-a070-66edf8ef5e46">

In local development, see http://127.0.0.1:8000/extending-phpunit.html#id2:

<img width="1305" alt="CleanShot 2023-05-29 at 14 02 12@2x" src="https://github.com/sebastianbergmann/phpunit-documentation-english/assets/605483/09a18984-0264-466a-bb66-f380bcefa152">

Do you know what I need to do here to show numbered captions, @sebastianbergmann?

Reply:
Works for me:
```
$ make html                    
sphinx-build -b html -d build/doctrees   src build/html
Running Sphinx v5.3.0
loading pickled environment... failed
failed: build environment version not current
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 15 source files that are out of date
updating environment: [new config] 15 added, 0 changed, 0 removed
reading sources... [100%] writing-tests-for-phpunit                                                                                                                                                                                                                                                                                                                                          
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] writing-tests-for-phpunit                                                                                                                                                                                                                                                                                                                                           
generating indices... genindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
```
![grafik](https://github.com/sebastianbergmann/phpunit-documentation-english/assets/25218/9b3a56cd-0700-4f21-b3c4-6343f5485b85)

You likely have different versions of Sphinx and/or the template we use compared to what I have locally and what ReadTheDocs.org uses. `src/requirements.txt` lists Sphinx 5.0.2 and sphinx_rtd_theme 1.2.0 as the versions to be used.